### ISSUE TITLE
Doesn't wortk with Firefox 64

### ISSUE THREAD

**Issue by d2r2r2** [downthemall/downthemall#305]
Doesn't wortk with Firefox 64 but only with Firefox 32.

Windows 20H2 (version 19042.685)

**Comment by d7415** [comment]
Do you mean 64bit and 32bit? If so, which version(s) of Firefox? What's not working?

**Comment by d2r2r2** [comment]
Yes, 64 bits: Firefox 84.0.1 (64-bit last updated today) with Windows 10 20H2 (last system version 19042.685). The same with the previous version of Windows 10 64bit.
It works well with Firefox 32bit but not with 64 bits: no backup, nothing at all.
I have already used Downthemall, before you stopped it for a while, with Firefox 32bit (with Windows 8.1 32-bit), but now I have a new 64-bit computer and the latest version of Windows 10 64 bits of update 19042.685. And Firefox 32bit has a lot of bugs with Windows 10 20H2 64bit.
Sorry for my bad English: I am French

**Comment by w1nt3ry** [comment]
Confirmed, I have the same problem with Firefox 64-bit.

Firefox version: 84.0.2 (64-bit)
Windows version: version 2004 (OS Build 19041.746)

When using DownThemAll, selecting downloads, then clicking on the Download button, nothing gets added to the Manager queue. 

I also tried going to the Manager directly, selecting "Add Download" button, and entering a download URL (one that I checked is valid). When I click Download in the dialog, nothing gets added to the Manager's queue and nothing downloads.

**Comment by nmaier** [comment]
Sounds like #159
